### ISSUE TITLE
[Staging] Epi Field Report figures don't show on Emergency page

### ISSUE THREAD

**Issue by nanometrenat** [IFRCGo/go-frontend#1097]
## Issue
I created an Epi Field Report in staging however when I look at the associated emergency it still shows the old figure types (and not the epi figures)

i.e. Emergency Overview
AFFECTED--
INJURED--
DEAD--
MISSING--
DISPLACED--
ASSISTED--
LOCAL STAFF--
VOLUNTEERS--
EXPAT DELEGATES--
Source: Natalie Epi Field Report Sunday the Twelfth , 12/04/2020, 16:54

Whereas I was expecting
Cases (Ministry of Health):
56
Suspected Cases (Ministry of Health):
22
Probable Cases (Ministry of Health):
11
Confirmed Cases (Ministry of Health):
99

etc.

## URL
https://ifrc-go-feature-1004-epi-field-report.surge.sh/emergencies/3938#details

## Steps to reproduce
Submit an epi Field Report and check the numbers pull through

![image](https://user-images.githubusercontent.com/33625192/79071967-cb15aa00-7cde-11ea-9c9d-1e00a24098ad.png)

![image](https://user-images.githubusercontent.com/33625192/79071977-da94f300-7cde-11ea-8719-c9efab029bd9.png)

## Expected behaviour
I was expecting to see the Epi numbers

## Related feature
#1004 #1096

## Criticality/Urgency
Not a blocker for the release but should be assessed as to priority given current focus on epi.

**Comment by nanometrenat** [comment]
@LukeCaley same with this re presumably being carried forward.  Not sure how you want to tag these to make sure they're not missed

**Comment by GregoryHorvath** [comment]
@mmusori @nanometrenat Anything to change?
![image](https://user-images.githubusercontent.com/30079559/80228576-7fa0bb80-864f-11ea-9425-160aebdc62d4.png)

cc @batpad depends on: https://github.com/IFRCGo/go-api/commit/f1aea58e350f0c716e5e5a8195d43f908d9cffc9

**Comment by mmusori** [comment]
After the above developments. The numeric details are not showing in the email notification and the emergency page.
![image](https://user-images.githubusercontent.com/7780809/80366964-85361580-888a-11ea-8007-3855007ac393.png)
The below is what comes out on the emergency page
![image](https://user-images.githubusercontent.com/7780809/80367123-cdedce80-888a-11ea-98d1-b32e07f747b4.png)

And this is the report generated 
![image](https://user-images.githubusercontent.com/7780809/80367057-ad257900-888a-11ea-9ca1-62e2897a18b5.png)

**Comment by GregoryHorvath** [comment]
@mmusori It seems you only have MoH selected for the topmost value and all the others doesn't have anything selected.
@batpad What was the base idea for the workflow here?

**Comment by batpad** [comment]
@mmusori yes, I believe this is related to the existing bugs around selecting sources and numerical details: https://github.com/IFRCGo/go-frontend/issues/1102 https://github.com/IFRCGo/go-frontend/issues/987

Apologies these are not fixed yet - this is an old bug that if a source is not selected, it does not save the numeric data. I will be focusing on these bugs this week, but I do believe that is not related to the changes for showing data on the emergency page.

**Comment by mmusori** [comment]
@GregoryHorvath you are right. So if you do not choose a source then the figures do not get saved? @batpad is it possible to make it mandatory to select a source if a figure is selected. The health team would not want figures without a source.

**Comment by mmusori** [comment]
> @mmusori yes, I believe this is related to the existing bugs around selecting sources and numerical details: #1102 #987
> 
> Apologies these are not fixed yet - this is an old bug that if a source is not selected, it does not save the numeric data. I will be focusing on these bugs this week, but I do believe that is not related to the changes for showing data on the emergency page.

If this is a know bug then it is not related. Just noticed it after the release.

**Comment by mmusori** [comment]
> @mmusori @nanometrenat Anything to change?
> ![image](https://user-images.githubusercontent.com/30079559/80228576-7fa0bb80-864f-11ea-9425-160aebdc62d4.png)
> 
> cc @batpad depends on: [IFRCGo/go-api@f1aea58](https://github.com/IFRCGo/go-api/commit/f1aea58e350f0c716e5e5a8195d43f908d9cffc9)

Is it possible not to have the figures circled in red for Epi.
![image](https://user-images.githubusercontent.com/7780809/80368184-9a13a880-888c-11ea-9bbb-dfb1a4b3b720.png)

**Comment by mmusori** [comment]
Feedback from Health:

EPI Key Figures on Emergency pages
1.	Remove old “Key Figures” for Epidemic Emergency pages: Numbers for “Affected”, “Injured”, “Dead”, “Missing”, and “Displaced” should not show on an Epidemic Emergency page, the same way that the numbers are no longer collected in the Epidemic Field Report.  In its place should be the “Epidemic Figures” (see screenshot below).  No comment on non-EPI key figures (e.g. “Assisted”, etc).

 
![image](https://user-images.githubusercontent.com/7780809/80379972-24650800-889f-11ea-99ed-3e1100eadc42.png)


2.	Epidemic Key Figure Source in header: There should not be multiple sources for Epidemic key figures (see logic above), but rather one source.  This one source can be cited, in parentheses following the Header text (e.g. Epidemic Figures (WHO) – screenshot above) or within the italicized “source” text below the key figures (e.g. Source: WHO, 4-27-2020). 
3.	“Confirmed”, “probable”, and “suspect” as sub-classification of “cases”: As in the form design, indent “Confirmed cases”, “probable cases” and “Suspect cases” to visually show that they are a sub-classification of all “Cases”
4.	Source timestamp: I do not understand why there is a timestamp affiliated with “Source”.  “Date of Data” is a DD/MM/YYYY value field with no affiliated timestamp.  I am currently concerned that it is pulling Field Report submission timestamp for “Date” rather than the manual “Date of Data” value field.  Do not want to show timestamp.

**Comment by GregoryHorvath** [comment]
@nanometrenat  After having a call with @mmusori and Roxanne we decided to hide the EPI data from the Emergency page until the whole logic behind it is reworked (Cases would a SUM of the other fields, only 1 selectable source, etc). @LukeCaley Felt like this is getting something urgent to have fixed (https://github.com/IFRCGo/go-frontend/issues/987).

@batpad @necoline Can we have a call about it so I could help with the redesign with full force? The Emergency page's figures and the email notifications are easy ones, I'm just not sure how **should** we tackle the backend part and the 'Create Field Report' ones. Like merging all the different figure fields in the DB into 1 or keep them separately, SUM the `...cases_XY` fields to the `...cases` one(s) on the backend vs frontend, etc?

**Comment by nanometrenat** [comment]
@GregoryHorvath @mmusori I am aware there are several tickets around Epi figures - but just to remind that on both Emergency page and submitted FR page as well as whatever the considerations are around Epi figures (Case numbers) it's also important that the standard figs around Assisted and Staff/Voly figs are also shown.

**Comment by GregoryHorvath** [comment]
@nanometrenat Oh yea thanks for the heads-up, those are not forgotten also!

**Comment by mmusori** [comment]
This is ready for implementation. 
![image](https://user-images.githubusercontent.com/7780809/82037111-fefb3b00-96a1-11ea-85f3-5eb9c9e807d1.png)
The logic issue will be addressed in Phase 4.3.

**Comment by batpad** [comment]
@GregoryHorvath @mmusori has any follow-up work from this been captured in separate tickets? If yes, is this good to close?

**Comment by GregoryHorvath** [comment]
@batpad Oh yes, I believe this can be closed, but @mmusori should confirm for sure.

**Comment by nanometrenat** [comment]
Closing due to elapsed time and many other tickets since.  @mmusori please re-open if work still needed here.
